### ISSUE TITLE
Inconsistency in providing aria-disabled for different components

### ISSUE THREAD

**Issue by AkshanshSrivastava03** [OfficeDev/office-ui-fabric-react#1140]
Some components provide us the "aria-disabled" props to specify (ChoiceGroup), whereas some provide us "inputProps" (Checkbox). 
Can't this be handled at fabric code side, ex: if "disabled" is set, use the same for aria-disabled internally by the fabric code for that component?

### Bug Report

- __Package version(s)__: 1.7.2
- __Browser and OS versions__:No

Requested priority: (Blocking, High, Normal, Low) Normal

#### Describe the issue:

Provide a consistent interface for handling aria-disabled for each component

#### Actual behavior:
Inconsistent 

#### Expected behavior:
Should be handled internally by the fabric code

**Comment by battletoilet** [comment]
Hey @AkshanshSrivastava03, we recognize that there is an inconsistency in how consumers interact with disabling input fields. We are currently working on fixing this issue #486 and should have something in pretty soon.

**Comment by oengusmacinog** [comment]
@AkshanshSrivastava03 we are now actively addressing this issue actively in our updates.  The pattern will still be inconsistent between certain modules because we need to avoid breaking changes, but we are working to consolidate native props and apply these standards.

I am going to close this Issue for now.  Thank you for bringing this to our attention so it could be addressed in our standards.
